tutorials/tts/FastPitch_Speaker_Interpolation.ipynb

```diff
@@ -0,0 +1,436 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9c4fd212",
+   "metadata": {},
+   "source": [
+    "# FastPitch Speaker Interpolation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd7e248e",
+   "metadata": {},
+   "source": [
+    "In this tutorial we will explore the following:\n",
+    "- What is speaker interpolation in a multi-speaker Text-to-Speech model\n",
+    "- Why we need speaker interpolation\n",
+    "- Step-by-step demo of performing speaker interpolation using pretrained multi-speaker FastPitch model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d1a78155",
+   "metadata": {},
+   "source": [
+    "## License\n",
+    "\n",
+    "> Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.\n",
+    ">\n",
+    "> Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "> you may not use this file except in compliance with the License.\n",
+    "> You may obtain a copy of the License at\n",
+    ">\n",
+    ">     http://www.apache.org/licenses/LICENSE-2.0\n",
+    ">\n",
+    "> Unless required by applicable law or agreed to in writing, software\n",
+    "> distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "> See the License for the specific language governing permissions and\n",
+    "> limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0d54ff5e",
+   "metadata": {},
+   "source": [
+    "### What is Speaker Interpolation in a Multi-speaker Text-to-speech Model\n",
+    "\n",
+    "Given a Multi-speaker Text-to-Speech (TTS) model that can generate speech in more than one speaker's voice, Speaker Interpolation is the process of synthesizing new synthetic **voices by combining two or more existing speaker voices** without the need for any further finetuning. Before we go in depths of the process let's understand why we need speaker interpolation?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "803e0e7c",
+   "metadata": {},
+   "source": [
+    "### Why we need speaker interpolation\n",
+    "\n",
+    "To release any TTS model, trained on datasets where the speakers do not want speech to be generated in their voice, speaker interpolation is important. This technique can be used to replace the original speaker's in the pretrained model with the interpolated synthetic speakers. This work would solve this problem.\n",
```

Comment:
If someone wants to release a model with only interpolated voices for privacy reasons, do we need steps for how to erase all of the original speaker embeddings from the model/checkpoint?

Maybe we can decouple the embeddings from the model by importing/exporting them from a separate file. We probably need the same functionality anyways for eventually utilizing external embeddings eg. from TitaNet.